src/index.coffee

```diff
@@ -5,9 +5,36 @@ httpProxy = require 'http-proxy'
 {version} = require('./package')
 
 CONFIG_DIR="#{process.env.HOME}/.mehserve"
+HTML_DIR="#{__dirname}/html"
 PORT = process.env.PORT ? 12439
 SUFFIXES=[/\.dev$/i, /\.meh$/i, /(\.[0-9]+){2,4}\.xip\.io$/i]
 
+handleError = (req, res, next) ->
+  (error) ->
+    title = null
+    message = null
+    code = null
+    switch error.code
+      when 'ECONNREFUSED'
+        code = 502
+        title = 'Bad Gateway'
+        message = "Looks like you forgot to run server on port #{error.port}!"
+      else
+        code = 500
+        title = 'Internal Server Error'
+        message = error.message || "Something bad happened!"
+
+    res.statusCode = code
+    content = fs.readFileSync("#{HTML_DIR}/error.html", 'utf8')
+      .split('{{code}}').join(code)
+      .split('{{title}}').join(title)
+      .replace('{{message}}', message)
+      .replace('{{version}}', version)
+      .replace('{{details}}', error.stack)
```

Comment:
Love your dependency-free template language! For consistency, please use the `.split().join()` mechanic for all the replacements - otherwise people updating the template and referencing `{{message}}` in multiple places may not get what they expect. Alternatively you could do a slightly more efficient version using a RegExp, something like:

```
templateVariables = {code, title, message, version, details}
content = fs.readFileSync(("#{HTML_DIR}/error.html", 'utf8')
.replace /\{\{([a-zA-Z0-9_-]+)\}\}/g, (_, varName) ->
    templateVariables[varName] ? ""
```